ONSdigital/florence
# Remove data vis split

### What

Remove the artifical split put in place through Florence to separate the creating, editing and publishing of visualisations. This is so that visualisations can be previewed before publication (if they are in the same collection).

### How to review

1) Switch this branch on Florence and 'feature/data-viz-preview' on Zebedee. 
2) A quick code check for sanity would be good.
3) Upload a new visualisation.
4) Embed that visualisation in a bulletin, in the same collection.
5) You should be able to see the unpublished visualisation on preview.

### Who can review

Probably @ian-kent or @jondewijones
